package.json

```diff
@@ -26,10 +26,7 @@
     "prepare": "npm run snyk-protect"
   },
   "lint-staged": {
-    "*.{js,ts,json,scss}": [
-      "prettier --write",
-      "git add"
-    ]
+    "*.{js,ts,json,scss}": ["prettier --write", "git add"]
```

Comment:
Looks like this file got reformated weirdly.

Looks like some kind of fight between npm and prettier.
Is this something you can try to fix ? Thanks.

Reply:
Sure will do

Reply:
Sadly prettier is stronger than me 😢